### DISCUSSION TITLE
different images between overview and Digit ROI

### DISCUSSION THREAD

**Discussion by nyunity** [jomjol/AI-on-the-edge-device#2667] [Q&A]
Hi all,

i created my esp with cam and think my numbers are good to read. But the AI reads the wrong position and give me the wrong value.

I made two pictures, maybe you can help me or have some idea?

<a href="https://ibb.co/71DzTrR"><img src="https://i.ibb.co/71DzTrR/Screenshot-2023-10-23-201544.jpg" alt="Screenshot-2023-10-23-201544" border="0"></a>

<a href="https://ibb.co/Lg8KypV"><img src="https://i.ibb.co/Lg8KypV/Screenshot-2023-10-23-201637.jpg" alt="Screenshot-2023-10-23-201637" border="0"></a>

thank you very much!

**Comment by Eisbahn** [comment]
I'm not an expert, but some suggestions:
- put the cam nearer to the digits/zoom in
- your boxes for the ROI seem to be way too big, adjust them to the real size
- is the cam twisted in the counter? In best case you deon't need to rotate the image

**Comment by schneich** [comment]
Hi @nyunity,

where are your alignment marker? (e.g. https://github.com/jomjol/AI-on-the-edge-device/discussions/1929#discussioncomment-4810985)
Those markers do not have to be very sophisticated. A simple sticker with a dot is enough.

Chris

**Reply by nyunity** [reply]
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/8504397/bb22e203-a6a5-4f6a-99f3-a8ec1b4f9dbd)

Thats mine stickers

**Comment by nyunity** [comment]
I get over mqtt the right value, after a few pictures, I get the wring values, because the image isnt correctly rotated... I will try to move the cam a) without angel and b) a little bit nearly.
